tests/bluetooth/shell/prj.conf

```diff
@@ -17,6 +17,7 @@ CONFIG_BT_TINYCRYPT_ECC=y
 CONFIG_CONSOLE_SHELL=y
 CONFIG_BT_SHELL=y
 CONFIG_BT_DEVICE_NAME="test shell"
+CONFIG_BT_DEVICE_NAME_DYNAMIC=y
 CONFIG_BT_DEVICE_NAME_MAX=28
```

Comment:
I guess this second line can be removed since 28 is now the default. Same goes btw for any other samples that use DEVICE_NAME_MAX. Don't you e.g. need to update the peripheral app's prj.conf?